<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Form</class>
 <widget class="QWidget" name="Form">
  <property name="enabled">
   <bool>true</bool>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>532</width>
    <height>345</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QHBoxLayout" name="horizontalLayout">
   <property name="spacing">
    <number>3</number>
   </property>
   <property name="margin">
    <number>0</number>
   </property>
   <item>
    <layout class="QVBoxLayout" name="mainInfoLayout">
     <property name="spacing">
      <number>3</number>
     </property>
     <property name="margin">
      <number>0</number>
     </property>
     <item>
      <layout class="QHBoxLayout" name="envFilterLayout">
       <property name="spacing">
        <number>3</number>
       </property>
       <property name="margin">
        <number>0</number>
       </property>
      </layout>
     </item>
     <item>
      <widget class="Line" name="line">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
      </widget>
     </item>
     <item>
      <layout class="QHBoxLayout" name="selectorLayout">
       <property name="spacing">
        <number>3</number>
       </property>
       <property name="margin">
        <number>0</number>
       </property>
       <item>
        <layout class="QVBoxLayout" name="fileSelectorLayout">
         <property name="spacing">
          <number>3</number>
         </property>
         <property name="sizeConstraint">
          <enum>QLayout::SetMinimumSize</enum>
         </property>
         <property name="margin">
          <number>0</number>
         </property>
        </layout>
       </item>
       <item>
        <layout class="QVBoxLayout" name="versionSelectorLayout">
         <property name="spacing">
          <number>3</number>
         </property>
         <property name="sizeConstraint">
          <enum>QLayout::SetMaximumSize</enum>
         </property>
         <property name="margin">
          <number>0</number>
         </property>
        </layout>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QVBoxLayout" name="pathEditLayout">
       <property name="spacing">
        <number>3</number>
       </property>
       <property name="margin">
        <number>0</number>
       </property>
      </layout>
     </item>
     <item>
      <widget class="QDialogButtonBox" name="buttonBox">
       <property name="standardButtons">
        <set>QDialogButtonBox::Cancel|QDialogButtonBox::Open</set>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QVBoxLayout" name="previewInfoLayout">
     <property name="spacing">
      <number>3</number>
     </property>
     <property name="sizeConstraint">
      <enum>QLayout::SetMinimumSize</enum>
     </property>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
